### PR TITLE
Adds start_all_apps action to existing actions

### PR DESCRIPTION
**Pull request by singhgarima**
Signed-off-by: Garima Singh <igarimasingh@gmail.com>

### CODE REVIEW

**Review by Lawouach — CHANGES_REQUESTED** [review]
LGTM

I like the examples you provide ;)

Could you add a changelog entry? :)

**Comment by singhgarima** [comment]
Added CHANGELOG entry as well 👍

File: chaoscf/actions.py [hunk 1]
```diff
@@ -61,6 +61,19 @@ def stop_app(app_name: str, configuration: Configuration, secrets: Secrets,
         path, configuration, secrets, method="PUT", body={"state": "STOPPED"})
 
 
+def start_all_apps(org_name: str, configuration: Configuration,
+                   secrets: Secrets):
+    """
+    Start application
```
Comment (Lawouach) [inline chaoscf/actions.py]: Sorry, I missed that bit earlier but would you mind making this docstring a little more useful? 

```
Start all applications in the given organization
```

**Review by Lawouach — APPROVED** [review]
LGTM

A slight change if you don't mind but that's not blocking. This would be nicer for users, specially when they use `chaos init`.

**Pull request merged**